scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/GradleBuildTool.scala

```diff
@@ -130,6 +130,24 @@ class GradleBuildTool(index: IndexCommand) extends BuildTool("Gradle", index) {
   private def initScript(toolchains: GradleJavaToolchains, tmp: Path): Path = {
     val agentpath = Embedded.agentJar(tmp)
     val pluginpath = Embedded.semanticdbJar(tmp)
+
+    val javacModuleOptions =
+      if (toolchains.isJavaAtLeast(9))
+        List(
```

Comment:
This list is available in `BuildInfo.javacModuleOptions` if you `.map(_.stripPrefix("-J"))`

Reply:
Nice!

The current problem is that the time at which this check is run is not right - I need to replicate it in the gradle script